embed.go

```diff
@@ -2,5 +2,8 @@ package jaegerclickhouse
 
 import "embed"
 
-//go:embed sqlscripts/*
-var EmbeddedFiles embed.FS
+//go:embed sqlscripts/no_replication/*
+var EmbeddedFilesNoReplication embed.FS
```

Comment:
Can you explain why are you moving the scripts? I don't see why it is needed.

Reply:
Yeah, it's not needed here, gonna revert this